README.md

```diff
@@ -1,3 +1,224 @@
-# proxy-chunks-minimal-example
+# How To Use Proxy & Chunks (Minimal Example)
 
-Example of how to use proxies and chunks
+Below is a summary of how to use proxies and chunking in Sway.
+
+**Proxies** refer to a blockchain design pattern that enables contract upgrades by forwarding calls made to the proxy contract to an implementation contract without changing the contract's address or state.
+
+**Chunking** addresses the issue of deploying contracts that exceed Sway's 100KB contract size limit. It splits the oversized contract into multiple blobs and uses a loader contract. When the loader contract is called upon, it automatically loads the blobs into memory, allowing the contract to function as normal.
+
+Both features are abstracted from developers and work out of the box with the latest version of `forc`. Below is a reference on how to use `Proxies`, `Chunking`, and how to combine them.
+
+## Prerequisites
+
+If you already have `fuelup` installed, run the commands below to make sure you are on the most up-to-date toolchain.
+
+```sh
+fuelup self update
+fuelup update
+fuelup default latest
+```
+
+At the time of writing you should be on `v0.63.3` or above
+
+```sh
+fuelup show
+```
+
+The command above should yield a similar output below
+
+```sh
+active toolchain
+----------------
+latest-aarch64-apple-darwin (default)
+  forc : 0.63.3
+    - forc-client
+      - forc-deploy : 0.63.3
+      - forc-run : 0.63.3
+    - forc-crypto : 0.63.3
+    - forc-debug : 0.63.3
+    - forc-doc : 0.63.3
+    - forc-fmt : 0.63.3
+    - forc-lsp : 0.63.3
+    - forc-tx : 0.63.3
+    - forc-wallet : 0.9.0
+  fuel-core : 0.34.0
+  fuel-core-keygen : 0.34.0
+
+fuels versions
+--------------
+forc : 0.66.1
+forc-wallet : 0.66.0
+```
+
+## Proxy
+
+> DO NOT ATTEMPT TO USE THE SRC-14 STANDARD OR UPGRADABILITY DIRECTLY YOURSELF TO AVOID COLLISION CATASTROPHES. THEY ARE FOR EDUCATIONAL PURPOSES ONLY. 
```

Comment:
I know I said something along these lines but looking at it like this it may be a bit strong. I think we want a tone that says we'd recommend you use our nice tools; they implement SRC14 which is intentionally minimal and usage of just the just the standard itself (or the examples in the sway-standards repo which are educational) is not recommended due to potential access control and storage collision issues

Reply:
Updated